model/crystal.h

```diff
@@ -1028,6 +1049,8 @@ namespace dxtbx { namespace model {
   protected:
     cctbx::sgtbx::space_group space_group_;
     cctbx::uctbx::unit_cell unit_cell_;
+    cctbx::uctbx::unit_cell postrefined_cell_;
```

Comment:
Slightly worried about the name `postrefined_cell` - this has a precisely defined meaning which we will not necessarily be using (i.e. using partiality to refine the cell post integration). `global_refined_cell` captures what we are doing, but I welcome opinions from others.

Reply:
the naming is hard, because what `dials.refine` does is global refinement. I figured that the only time we would set this is after some sort of post integration refinement. I thought that `postrefined` would capture this without being specific about the algorithm, but I appreciate the possibility for confusion. In dials/dials#670 we talked about a "best overall" cell, but I don't like this because it begs the question "best for what?"